basics/src/test/java/basics/LibraryTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package basics;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class LibraryTest {
+
+    @Test
+    public void rollNotNull() {
+        Library classUnderTest = new Library();
+        assertNotNull("roll should return 'true'", classUnderTest.roll(1));
+    }
+
+    @Test
+    public void rollLengthIsCorrect() {
+        Library classUnderTest = new Library();
+        assertTrue("roll should have a length of at least n", classUnderTest.roll(1).length == 1);
+    }
+
+    @Test
```

Comment:
in the future, id like to see more tests for each method. test some edge cases or at the very least test each method a couple times so i know its working completely and not just working in that single case that was tested